codec-http/src/main/java/io/netty/handler/codec/http/HttpRequestDecoder.java

```diff
@@ -90,6 +90,7 @@
  */
 public class HttpRequestDecoder extends HttpObjectDecoder {
 
+    private static final AsciiString Accept = AsciiString.cached("Accept");
```

Comment:
> We prefer the lower-case version as this is compatible with HTTP2 where all header names should be lowercase

The current approach is consistent with what we already have inside `HttpRequestDecoder` for other cached cases. In a follow-up, we can change all local constants in favor of `HttpHeaderNames`, as well as adding a check for `Accept` and `accept` we can receive from the wire.